dev.Dockerfile

```diff
@@ -1,7 +1,7 @@
 #############
 ## Stage 1 ##
 #############
-FROM ruby:2.6.6 as builder
+FROM ruby:3.0.3-bullseye as builder
```

Comment:
Could you see if there's 3.1 for these please. May as well jump to that if we can.